### PR TITLE
Vijayasree/fix: wrong market description for EU & UK

### PR DESCRIPTION
**Pull request by vijayasree-deriv**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/markets/components/sections/_hero.tsx [hunk 1]
```diff
@@ -88,9 +89,16 @@ export const Hero = () => {
...
-                    {localize(
-                        'Learn about the markets that you can trade online with Deriv, including forex, synthetic indices, stocks, stock indices, cryptocurrencies, basket indices, and commodities.',
-                    )}
+                    <EU>
+                        <Localize translate_text="Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, stocks, stock indices, and cryptocurrencies." />
+                    </EU>
+                    <UK>
+                        <Localize translate_text="Learn about the markets that you can trade online with Deriv, including forex, commodities, stocks, and stock indices." />
+                    </UK>
+
```
Comment (prince-deriv) [inline src/pages/markets/components/sections/_hero.tsx:98]: Nitpick remove the extra space

**Review by prince-deriv — APPROVED** [review]

File: src/pages/markets/components/sections/_hero.tsx [hunk 2]
```diff
@@ -88,9 +89,15 @@ export const Hero = () => {
                     align="center"
                     mt="1.6rem"
                 >
-                    {localize(
-                        'Learn about the markets that you can trade online with Deriv, including forex, synthetic indices, stocks, stock indices, cryptocurrencies, basket indices, and commodities.',
-                    )}
+                    <EU>
+                        <Localize translate_text="Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, stocks, stock indices, and cryptocurrencies." />
```
Comment (sean-binary) [inline src/pages/markets/components/sections/_hero.tsx:93]:
```suggestion
                        <Localize translate_text="Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, stocks & indices, and cryptocurrencies." />
```

Comment (sean-binary) [reply src/pages/markets/components/sections/_hero.tsx:93]: `stocks, stock indices` to be changed to `stocks & indices` for all 3. @vijayasree-deriv

Comment (vijayasree-deriv) [reply src/pages/markets/components/sections/_hero.tsx:93]: will confirm with @shen.

Comment (vijayasree-deriv) [reply src/pages/markets/components/sections/_hero.tsx]: updated as per the redmine card @sean-binary

**Review by prince-deriv — APPROVED** [review]

**Review by sean-binary — DISMISSED** [review]

**Review by prince-deriv — APPROVED** [review]

**Review by mohammad-rework — APPROVED** [review]

**Pull request merged**